### PR TITLE
Fix more covs

### PR DESCRIPTION
**Pull request by dav1901**
Fix remaining defects from #12026.

### CODE REVIEW

File: libr/anal/p/anal_java.c [hunk 1]
```diff
@@ -666,22 +666,24 @@ static int java_analyze_fns( RAnal *anal, ut64 start, ut64 end, int reftype, int
...
-				RAnalFunction *fcn = r_anal_fcn_new ();
-				fcn->cc = r_str_const (r_anal_cc_default (anal));
-				java_set_function_prototype (anal, fcn, method);
-				result = analyze_from_code_attr (anal, fcn, method, loadaddr);
-				if (result == R_ANAL_RET_ERROR) {
-					//eprintf ("Failed to parse java fn: %s @ 0x%04"PFMT64x"\n", fcn->name, fcn->addr);
-					return result;
-					// XXX - TO Stop or not to Stop ??
+			if (method) {
+				if ((analyze_all) ||
```
Comment (radare) [inline libr/anal/p/anal_java.c]: why this double (())

**Comment by radare** [comment]
there are zillions of XX, can you rebase?

File: libr/flag/flag.c [hunk 2]
```diff
@@ -562,6 +562,7 @@ R_API RFlagItem *r_flag_set(RFlag *f, const char *name, ut64 off, ut32 size) {
 				r_skiplist_insert (f->by_off, flagsAtOffset);
 			}
 		}
+		R_FREE (flagsAtOffset);
```
Comment (radare) [inline libr/flag/flag.c]: if you remove this in here it will fuck the skiplist, that's why there are tons of XX

Comment (dav1901) [reply libr/flag/flag.c]: Changes reverted

**Pull request merged**